tests/data/vision/pgm_input_data_en.json

```diff
@@ -6,49 +6,52 @@
     ],
   "sym_load":
     [
-      {"id": 13, "node": 12, "id_reference": {"table": "Transformer loads", "name": "load", "key": {"Node.Number": 3, "Subnumber": 4}}, "Name": "transformer_load_1"},
-      {"id": 19, "node": 2, "id_reference": {"table": "Loads", "key": {"Node.Number": 3, "Subnumber": 6}}, "Name": "load1"}
+      {"id": 15, "node": 13, "id_reference": {"table": "Transformer loads", "name": "load", "key": {"Node.Number": 3, "Subnumber": 4}}, "Name": "transformer_load_2"},
+      {"id": 16, "node": 14, "id_reference": {"table": "Transformer loads", "name": "load", "key": {"Node.Number": 3, "Subnumber": 2}}, "Name": "transformer_load_1"},
+      {"id": 22, "node": 2, "id_reference": {"table": "Loads", "key": {"Node.Number": 3, "Subnumber": 6}}, "Name": "load1"}
     ],
   "sym_gen":
     [
-      {"id": 14, "node": 12, "id_reference": {"table": "Transformer loads", "name": "generation", "key": {"Node.Number": 3, "Subnumber": 4}}, "Name": "transformer_load_1"},
-      {"id": 15, "node": 12, "id_reference": {"table": "Transformer loads", "name": "pv_generation", "key": {"Node.Number": 3, "Subnumber": 4}}, "Name": "transformer_load_1"},
-      {"id": 17, "node": 3, "id_reference": {"table": "Synchronous generators", "key": {"Node.Number": 4, "Subnumber": 2}}, "Name": "syngen1"},
-      {"id": 18, "node": 2, "id_reference": {"table": "Wind turbines", "key": {"Node.Number": 3, "Subnumber": 3}}, "Name": "wind1"},
-      {"id": 23, "node": 2, "id_reference": {"table": "Pvs", "key": {"Node.Number": 3, "Subnumber": 1}}, "Name": "pv1"}
-    ],
-  "three_winding_transformer":
-    [
-      {"id": 24, "node_1": 1, "node_2": 2, "node_3": 4, "status_1": 1, "status_2": 1, "status_3": 1, "u1": 11000.0, "u2": 400.0, "u3": 400.0 , "sn_1": 30000000.0, "sn_2": 40000000.0, "sn_3": 35000000.0, "uk_12": 0.01, "uk_13": 0.02, "uk_23": 0.03, "pk_12": 10000.0, "pk_13": 20000.0, "pk_23": 30000.0,  "i0": 0.00171428, "p0": 60000.0, "winding_1": 2, "winding_2": 1, "winding_3": 1, "clock_12": 5, "clock_13": 7, "tap_pos": 2, "tap_side": 2, "tap_min": -7, "tap_max": 8, "tap_nom": 2, "tap_size": 10.0, "id_reference": {"table": "Three winding transformers", "key": {"Number": 1}}}
+      {"id": 17, "node": 14, "id_reference": {"table": "Transformer loads", "name": "generation", "key": {"Node.Number": 3, "Subnumber": 2}}, "Name": "transformer_load_1"},
+      {"id": 18, "node": 14, "id_reference": {"table": "Transformer loads", "name": "pv_generation", "key": {"Node.Number": 3, "Subnumber": 2}}, "Name": "transformer_load_1"},
+      {"id": 20, "node": 3, "id_reference": {"table": "Synchronous generators", "key": {"Node.Number": 4, "Subnumber": 2}}, "Name": "syngen1"},
+      {"id": 21, "node": 2, "id_reference": {"table": "Wind turbines", "key": {"Node.Number": 3, "Subnumber": 3}}, "Name": "wind1"},
+      {"id": 26, "node": 2, "id_reference": {"table": "Pvs", "key": {"Node.Number": 3, "Subnumber": 1}}, "Name": "pv1"}
```

Comment:
@mgovers  The difference can be seen here, check the `Name`: transformer_load_2 is not visible in the sym_gens